test/drop_test/action.py

```diff
@@ -104,6 +105,17 @@ async def run(self, drop: ffi.Drop):
         raise Exception(f"Action must have thrown DropException")
 
 
+class ExpectAnyError(Action):
+    def __init__(self, action: Action):
+        self._action = action
+
+    async def run(self, drop: ffi.Drop):
+        try:
+            await self._action.run(drop)
+        except:
+            pass
```

Comment:
This is more like `ExpectAnyErrorOrNoError`, it should fail if no error was produced I assume?

Reply:
Oh yeah, let me fix this